src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs

```diff
@@ -98,28 +98,13 @@ public bool IsSelected
             }
         }
 
-        public bool IsInput
+        public bool? IsInputNode
```

Comment:
just seems a bit confusing to have the same property name on both node model and view, not a big deal I guess, since it's public an we add a comment here?